src/renate/benchmark/experimentation.py

```diff
@@ -179,6 +180,10 @@ def execute_experiment_job(
         deterministic_trainer: When true the Trainer adopts a deterministic behaviour also on GPU.
             In this function this parameter is set to True by default.
         job_name: Name of the experiment job.
+        strategy: String denoting lightning distributed strategy.
+        precision: String for which precision to use.
+        retain_intermediate_state: Flag to retain intermediate models and buffer states.
```

Comment:
maybe rephrase to "retain states after each update"? otherwise very redundant description and not helpful for anyone who doesn't understand what "intermediate" refers to.